docs/docs/install/docker-compose/troubleshooting.md

```diff
@@ -1,7 +1,8 @@
-# Troubleshooting
+# Troubleshooting Docker Compose Installation
 
 ## Restarting installation
-If you need to re-start installation for any reason, please do not re-run the installation script! You'll need to run the [uninstall steps](./uninstall.md) before trying to run the installation script again.
+
+If you need to re-start installation for any reason, do not re-run the installation script! You'll need to run the [uninstall steps](./index.md#uninstall) before trying to run the installation script again.
```

Comment:
This is not strictly accurate (though I understand you didn't change it). They can, in fact, safely re-run the script up till the point where the script actually starts the Docker container infrastructure. Even then it's only necessary to remove the volumes via 
```sh
docker compose -f /etc/mathesar/docker-compose.yml --profile prod down -v
```
(Note the lack of `--rmi all` above.)

The reason to avoid having them perform the uninstall steps is that those steps will fail and throw errors if they haven't gotten to where they actually have the config files and are running some Docker containers. This may give them the impression that now they're stuck and can't install, when they could have simply re-run the script.

For example, if they fat-finger a host name and realize it a step or two later, they can safely CTRL+C and restart. This is a feature of the installer that is useful (IMO).

Reply:
Because this is an unrelated content change, I'd prefer to handle this separately from this PR. I'd make the edit myself, but it seems that you would be better suited to make the edit yourself, @mathemancer. I don't want to get anything incorrect about the edit and further delay this PR since I think there are some useful improvements to the installation content in this PR. Maybe you could open a separate PR for these changes, even just merging it yourself?